frontend/compositions/officer-view/optional-officer-info/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { OfficerRecordType } from "../../../models/officer"
+import styles from "./optional-officer-info.module.css"
+
+export default function OptionalOfficerInfo(officer: OfficerRecordType) {
+  const { birthDate, gender, race, ethnicity, incomeBracket } = officer
+  const { category, data } = styles
+
+  const dateString: string = new Date(birthDate).toLocaleDateString().split(",")[0]
+  return (
+    <div>
+      <div style={{ display: "flex" }}>
```

Comment:
css style modules here too please. since they are TS files should be easily sharable